spec/browser/exceljs.spec.js

```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-require('babel-polyfill');
+require('@babel/polyfill');
```

Comment:
unluckily, this break backward compatibility :(

Reply:
I'm not hard against breaking changes - we can bump major version for that.
I've never been particularly happy with the PromiseLib compromise - what happened was, the original promise lib was Bluebird, which was ok, but when I discovered how many megabytes it used it really hurt the browser bundle, that's when Promish was introduced. But the switch broke some clients so the option to inject the Promise lib of choice was added.
Now that native promise supports "finally" (as of Node v10.3) it might be time to fix it properly.
Since it would be a breaking change, we should announce it properly, give users time to adjust, etc

Reply:
Fun fact: we don't even need `Promise.finally` anymore, since with async/await it's just `try/catch/finally`.

@guyonroche I think it's not that big a change, even our tests are passing without any real modifications to them. I think this change doesn't need a huge amount of forewarning, unless there's other stuff that you'd like to clean up for exceljs v2?

Reply:
I've reverted this change, this PR now uses the version of babel-polyfill that is currently on `master`